src/MiddlewareListener.php

```diff
@@ -1,25 +1,26 @@
 <?php
 /**
- * @link      http://github.com/zendframework/zend-mvc for the canonical source repository
- * @copyright Copyright (c) 2005-2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @see       https://github.com/zendframework/zend-mvc for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
Please revert this. (The current year is only needed for _new_ files.)

Reply:
@froschdesign 
I didn't know that. Can you confirm the following, just to be clear for future updates?
1. If a file is edited and the ending year of the range is not the current year, it must be updated (e.g. 2005-2017 becomes 2005-2018)
2. If a file is edited and there is no ending year, the current year must be added at the end (e.g. 2017 becomes 2017-2018)
3. For new files the current year must be added (2018)